icons/icloud.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>icloud icon</title><path d="M13.679 5c2.72 0 5.681 2.016 5.607 6.365 5.49.643 4.409 7.635.114 7.635h-14.508c-4.124 0-5.751-6.252-1.105-8.04-.459-2.491 2.479-4.366 4.564-2.187.477-1.023 1.885-3.773 5.328-3.773m0-1c-2.418 0-4.529 1.35-5.674 3.36-2.238-1.443-5.195.194-5.214 2.975-4.72 2.32-3.101 9.665 2.101 9.665h14.508c5.582 0 6.32-8.406.854-9.466-.141-3.633-3.031-6.534-6.575-6.534z"/></svg>
```

Comment:
"icloud" should be "iCloud"

Also the icon they officially use is filled - changing to a stroked version doesn't accurately represent the brand.

I took the liberty of making an updated version of the logo that more accurately represents the official one:

<p align="center"><img src="https://gistcdn.githack.com/birjolaxew/8ec7c6e8cc1cce38d60876560de6dc3d/raw/a1c68400a947f99b663a0b1d4671ee13716775ea/icloud.svg" width="128" /></p>

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>iCloud icon</title><path d="M20.013 10.726l.001-.028A6.346 6.346 0 0 0 8.09 7.67a3.414 3.414 0 0 0-4.989 2.829A4.72 4.72 0 0 0 0 14.929a4.72 4.72 0 0 0 4.719 4.719h14.807a4.475 4.475 0 0 0 .487-8.922z"/></svg>
```

Feel free to use this one as a replacement for the one in the PR